CONTRIBUTING.md

```diff
@@ -37,12 +37,49 @@ See the [XAITK-Saliency README](README.md) file for additional information.
 
 XAITK-Saliency uses GitHub for code review and GitHub Actions for continuous
 testing.
-New pull requests trigger CI when the merge target is the `master` branch.
+New pull requests trigger Continuous Integration workflows (CI) when the merge
+target is the `master` or `release`-variant branch.
 All checks/tests must pass before a PR can be merged by an individual with the
 appropriate permissions.
 
 We use Sphinx for manual and automatic API [documentation](docs).
 
+### Jupyter Notebooks
+When adding or modifying a Jupyter Notebook in this repository, consider the
+following:
+* Notebooks should be executable on Google Colab.
+* Notebooks should be included in the appropriate CI workflow and be runnable
+  in that environment in a timely manner.
+
+#### Colab
+XAITK-Saliency example notebooks have established a pattern of providing a
+reference to, and support execution in, Google Colab.
+Notebooks should include, near the bottom of their introduction section, a
+reference to itself on Google Colab.
+See existing example notebooks for examples on what this looks like, e.g. at
+the bottom of cell #1 in the ``examples/OcclusionSaliency.ipynb`` notebook.
+
+Notebooks should also be runnable when executed in Google Colab.
+This often requires a cell that performs ``pip install ...`` commands to bring
+in the appropriate dependencies (including `xaitk-saliency` itself) into the
+at-first empty Colab environment.
+This will also help with execution in the CI environment as detailed next.
+
+#### Notebook CI
+This repository has set up a CI workflow to execute notebooks to ensure their
+continued functionality, avoid bit-rot and avoid violation of established
+use-cases.
+When contributing a Jupyter notebook, as an example or otherwise, a reference
+should be added to this CI workflow ([located here ~L27](
+.github/workflows/ci-example-notebooks.yml)) to enable its inclusion in the CI
+testing.
+
+To that end, in developing the notebook, consider its execution in this CI
+environment:
+* should be concise in its execution time to not stall or time-out the CI
+  workflow.
+* should be light on computational resources to not exceed.
```

Comment:
Seems like this bullet point is missing a word to complete the thought?

Reply:
Yup, I didn't finish that.